AutoRest/Generators/Java/Azure.Java.Tests/src/main/java/fixtures/paging/models/ProductResult.java

```diff
@@ -1,66 +0,0 @@
-/**
```

Comment:
Yes, it's fine!

Sorry it's my my bad not removing that file - looks like regenerate:expected:javaazure does not remove before regeneration.